theme/static/js/hs_file_browser.js

```diff
@@ -596,19 +596,23 @@ function bindFileBrowserItemEvents() {
             }
             menu = $("#right-click-menu");
 
-            var fileAggType = $(event.target).closest("li").find("span.fb-logical-file-type").attr("data-logical-file-type");
+            fileAggType = [];
```

Comment:
```var``` should be added for ```fileAggType``` here when it is first initialized here, right?

Reply:
yeah, that's a good catch.  Without var it is global.  I'll make the change